### ISSUE TITLE
[Bug]: Android Accessibility Unlabeled Buttons

### ISSUE THREAD

**Issue by shortwavesurfer2009** [simplex-chat/simplex-chat#3630]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Platform

Android

### OS version

Android 13

### App version

5.5-beta.0 (168)

### Current Behavior

The filter button to the right of "chats" and the qr code button to the right of "search or paste SimpleX link" read as "unlabeled" when using Talkback

### Expected Behavior

These buttons should be labeled

### Steps To Reproduce

1. go to system settings > Accessibility > aLTalkback and turn it on
2. Open simplex
3. Tap the buttons mentioned above

**Comment by shortwavesurfer2009** [comment]
This is still not fixed as of the latest release.
